lib/bolt/config.rb

```diff
@@ -117,6 +119,21 @@ def update_from_file(data)
           self[:transports][:winrm][:ca_cert] = data['winrm']['ca-cert']
         end
       end
+
+      if data['pcp']
+        if data['pcp']['service-url']
+          self[:transports][:pcp][:service_url] = data['pcp']['service-url']
+        end
+        if data['pcp']['ca-cert']
+          self[:transports][:pcp][:ca_cert] = data['pcp']['ca-cert']
+        end
+        if data['pcp']['token-file']
+          self[:transports][:pcp][:token_file] = data['pcp']['token-file']
+        end
+        if data['pcp']['environment']
```

Comment:
This should be task-environment I think. Sorry if I didn't update the ticket

Reply:
Actually we should probably refer to it even more specifically in the code.
```
pcp:
  task-environment: my_environment
```
is good in the config file but if it's going to be visible to all the transports it should probably be 'orch_task_environment' as a variable.